Samples/java/src/main/res/values/integers.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <!-- TODO: Change this buildingId to match the building on MaaS -->
-    <integer name="buildingId">12345</integer>
+    <integer name="buildingId">1234</integer>
```

Comment:
Can we use the same building id in both kotlin & java sample (12345). just for consistency